packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -437,6 +440,7 @@ describe('backupNg', () => {
         mode: jobInput.mode,
         reportWhen: jobInput.settings[''].reportWhen,
       },
+      infos: [{ data: { vms: [jobInput.vms.id] }, message: 'vms' }],
```

Comment:
I don't understand this change, AFAIR it was always the case… :thinking:

Reply:
This task wasn't logged when there is no VM concurrency.

Reply:
I see, IMHO the new behavior is better because more coherent, what do you think?

Reply:
I agree :+1: